### DISCUSSION TITLE
Bootstrap v5.0 SCSS: Duplicate CSS variables in :root

### DISCUSSION THREAD

**Discussion by teamunixcreativity** [twbs/bootstrap#35958] [Q&A] [answered]
I'm trying to add more colors to my `$theme-colors`  **SCSS** map it works but when I reload my page I find that it's **duplicated** in my `:root` look to the image below:

![Image](https://i.ibb.co/Z6WbzGc/1.png)

`_variables.scss`

```
...

$blue: #0d6efd;
$indigo: #6610f2;
$purple: #6f42c1;
$pink: #d63384;
$red: #dc3545;
$orange: #fd7e14;
$yellow: #ffc107;
$green: #198754;
$teal: #20c997;
$cyan: #0dcaf0;

$primary: #c55e0a;
$secondary: $gray-600;
$success: $green;
$info: $cyan;
$warning: $yellow;
$danger: $red;
$light: $gray-100;
$dark: $gray-900;

$theme-colors: (
     "primary": $primary,
     "secondary": $secondary,
     "success": $success,
     "info": $info,
     "warning": $warning,
     "danger": $danger,
     "light": $light,
     "dark": $dark
);

$custom-theme-colors: (
     "blue": $blue,
	 "indigo": $indigo,
	 "purple": $purple,
	 "pink": $pink,
	 "orange": $orange,
	 "teal": $teal,
);

$theme-colors: map-merge($theme-colors, $custom-theme-colors);

...
```

**how I can fix this issue?**

**Comment by WinterSilence** [comment]
@teamunixcreativity It's redundant values, use `$colors` instead. [To remove duplicates](https://github.com/twbs/bootstrap/pull/35967/files)

**Comment by mdo** [comment, marked as answer]
This is because we add our `$colors` to `:root` already, so when you merge them to `$theme-colors`, you're inevitably going to have duplication. As suggested, you may need to trim some other values from the `$colors` map if you go this route.
